cdc/model/sink.go

```diff
@@ -719,6 +748,115 @@ func (t *SingleTableTxn) GetCommitTs() uint64 {
 	return t.CommitTs
 }
 
+// TrySplitAndSortUpdateEvent split update events if unique key is updated
+func (t *SingleTableTxn) TrySplitAndSortUpdateEvent() error {
+	if len(t.Rows) < 2 {
+		return nil
+	}
+	newRows, err := convertRowChangedEvents(t.Rows)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	sort.Sort(RowChangedEvents(newRows))
+	t.Rows = newRows
+	return nil
+}
+
+// convertRowChangedEvents uses to convert RowChangedEvents to TableSinkRowChangedEvents.
```

Comment:
`TableSinkRowChangedEvents` sounds like a new term but not exist in the code base, how about remove it ?